docs/mine-hnt/full-hotspots/become-a-maker/maker-approval-auditing.mdx

```diff
@@ -27,3 +27,6 @@ The full process for approving and auditing new manafacturers is as follows
    time
 6. Once Makers pass hardware audit, they are allowed to proceed to Network Integration.
 7. The final stage of Network Integration is being able to add Hotspots to the Onboarding Server.
+
+Note: Helium does not specifically endorse or recommend makers, manufacturers or vendors, it approves 
+the hotspots the makers produce can operate on the network to HIP 19 requirements.
```

Comment:
Capitalize Hotspots. This should be two sentences.

Reply:
"it approves the hotspots the makers produce can operate"
It "verifies"? The way this sentence currently reads it implies that we approve that they operate which doesn't read right.